bazel/repositories.bzl

```diff
@@ -113,8 +113,8 @@ def _python_deps():
         build_file = "@envoy//bazel/external:pyyaml.BUILD",
     )
     native.bind(
-        name = "pyyaml",
-        actual = "@com_github_yaml_pyyaml//:pyyaml",
+        name = "yaml",
+        actual = "@com_github_yaml_pyyaml//:lib/yaml",
```

Comment:
I think the actual name should be `@com_github_yaml_pyyaml//:yaml`. The external workspace binds only top-level names.

Reply:
@kyessenov, thanks for checking. I guess that should be the directory after cloning from com_github_yaml_pyyaml defined in repository_locations.bzl. I made that change and couldn't fix build. Could be please take a look again? Thanks in advance!

Reply:
Hm, This name resolves to the `py_library` in `pyyaml.BUILD`. You can rename that library to `lib/yaml` or use `:yaml` here, either way is fine, I think.

Reply:
@kyessenov , changed with https://github.com/envoyproxy/envoy/pull/2957/commits/5602eed59593b896ac5d9a6a553222a83852f5d9

Reply:
@kyessenov, friendly ping. :)

Reply:
Sorry, missed notifications :( Bazel stuff looks fine to me.